bitz0x75/Twitter-Lite-Frontend
# #165408017 Setup testing Environment

#### What does this PR do?
- Sets up a testing environment for the react app
#### Description of Task to be completed?
 - add packages that are required for testing
#### How should this be manually tested?
- checkout to `setup-testing-environment-165408017` branch
- ensure the testing packages are setup (enzyme)
#### Any background context you want to provide?
- N/A
#### What are the relevant PT Stories?
[#165408017](https://www.pivotaltracker.com/story/show/165408017)
#### Screenshots (if appropriate)
- N/A
#### Questions:
N/A
